api/capi/src/nnstreamer-capi-single.c

```diff
@@ -333,8 +333,14 @@ ml_single_inference (ml_single_h single,
   }
 
   /* Try to get the result */
+#if (GST_VERSION_MINOR >= 10)
```

Comment:
Check MAJOR along with this?
(e.g., say FALSE for all 0.*, TRUE for 2.*)

Reply:
changed condition 
```
#if (GST_VERSION_MAJOR >= 1 && GST_VERSION_MINOR >= 10)
```